src/components/connectors/ConnectorTiles.tsx

```diff
@@ -150,8 +150,9 @@ function ConnectorTiles({
     const primaryCtaClick = (row: ConnectorWithTagDetailQuery) => {
         navigateToCreate(
             row.connector_tags[0].protocol,
-            row.connector_tags[0].id,
-            replaceOnNavigate
+            row.connector_tags[0].connector_id,
+            replaceOnNavigate,
+            row.connector_tags[0].protocol === 'capture'
```

Comment:
The `useEntityCreateNavigate` hook's fourth input parameter dictates whether the create form path is selected as the base for the next URL. At this point in time, the existing entity check must be hidden for captures. Clicking the _Capture_ CTA on a connector tile/card should always advance the user directly to the create form.